payroll/apps.py

```diff
@@ -39,6 +39,17 @@
     "csv_reconciliation_paid_no": "No",
     "payroll_delete_event": "payroll.payroll_delete",
     "opensearch_synch": False,
+
+    "gateway_base_url": "http://41.175.18.170:8070/api/mobile/v1/",
+    "endpoint_payment": "mock/payment",
+    "endpoint_reconciliation": "mock/reconciliation",
+    "payment_gateway_api_key": "default-api-key",
+    "payment_gateway_basic_auth_username": "soldevelo@coremis.com",
+    "payment_gateway_basic_auth_password": "password",
+    "payment_gateway_timeout": 5,
+    "payment_gateway_auth_type": "basic",  # can be 'token' or 'basic'
+    "payment_gateway_class": "payroll.payment_gateway.MockedPaymentGatewayConnector",
+    "receipt_length": 8
```

Comment:
Seems like **mock** credentials hardcoded in the config should not be a problem. However @lruzicki please take a look.